README.md

```diff
@@ -34,12 +32,12 @@ packages:
 
 To run:
 
-	# start up python
-	> python
+    # start up python
+    > python
         ... (startup messages) ...
-	>>> from pyds9 import *
-	>>> print ds9_targets()
-	>>> d = DS9()
+    >>> from pyds9 import *
+    >>> print ds9_targets()
+    >>> d = DS9()
```

Comment:
For me this currently doesn't work (#17).
Can you show just one more command here how to do something useful?
Then it would be a useful getting started example.

One possibility is that pyds9 ships and installs a very small FITS example image that's accessible via `pyds9.TEST_IMAGE` and used for this example in a load command?